### DISCUSSION TITLE
Usb

### DISCUSSION THREAD

**Discussion by JorchMantilla** [bottlesdevs/Bottles#290] [Q&A]
Hi everyone!
Can I use a USB stick from bottles?

**Comment by mirkobrombin** [comment]
Hi. Wine support external devices by default.

**Reply by JorchMantilla** [reply]
How?

**Reply by mirkobrombin** [reply]
Just link the device from winecfg (Drives tab)
![image](https://user-images.githubusercontent.com/9479903/120060169-6dfdc480-c056-11eb-8c05-cdbf57a6e555.png)
